pkg/provider/gcp.go

```diff
@@ -24,7 +23,7 @@ type GCPProvider struct {
 	Clust         string `survey:"cluster"`
 	Proj          string `survey:"project"`
 	bucket        string
-	region        string
+	Reg           string `survey:"region"`
 	storageClient *storage.Client
 	ctx           context.Context
```

Comment:
I think i must have been plumbing the context around since it was needed for google clients.  The field itself can likely be dropped and replaced with calls to `context.Background()`

Reply:
I was wanting to place a bucket region in there actually, since it requires a bit of parsing that will be annoying to do upstream.